utils/iscn.ts

```diff
@@ -26,7 +25,7 @@ export async function getSigningClient () {
   return client
 }
 
-export function formatISCNTxPayload (payload: ISCNRegisterPayload): ISCNSignPayload {
+export function formatISCNTxPayload (payload: any) {
```

Comment:
```suggestion
export function formatISCNTxPayload (payload: ISCNRegisterPayload  & any): ISCNSignPayload {
```
Please see if this works

Reply:
[✨ Update formatISCNTxPayload to include stakeholders](https://github.com/likecoin/nft-book-press/pull/321/commits/554665c10512d3c439e89c219b665b413affb665)